cmd/api/src/database/assetgrouptags.go

```diff
@@ -377,8 +377,8 @@ func (s *BloodhoundDB) UpdateAssetGroupTag(ctx context.Context, user model.User,
 		if !tag.Position.Valid {
 			return model.AssetGroupTag{}, fmt.Errorf("position is required for an existing tier")
 		}
-	} else if tag.Position.Valid || tag.RequireCertify.Valid {
-		return model.AssetGroupTag{}, fmt.Errorf("position and require_certify are limited to tiers only")
+	} else if tag.Position.Valid || tag.RequireCertify.Valid || tag.AnalysisEnabled.Bool {
+		return model.AssetGroupTag{}, fmt.Errorf("position, require_certify, analysis_enabled are limited to tiers only")
```

Comment:
- We do want to use `.Valid` as CR and @urangel mentioned because otherwise it is checking the value of `analysis_enabled` instead of whether or not it was specified. If we do want to allow it to be specified but only if it's false, then we should use `.ValueOrZero`
- Grammar for a list
```suggestion
	} else if tag.Position.Valid || tag.RequireCertify.Valid || tag.AnalysisEnabled.Valid {
		return model.AssetGroupTag{}, fmt.Errorf("position, require_certify, and analysis_enabled are limited to tiers only")
```